optimade/server/config.py

```diff
@@ -153,7 +242,10 @@ class ServerConfig(BaseSettings):
     )
     index_links_path: Path = Field(
         Path(__file__).parent.joinpath("index_links.json"),
-        description="Absolute path to a JSON file containing the MongoDB collection of /links resources for the index meta-database",
+        description=(
+            "Absolute path to a JSON file containing the MongoDB collection of /links resources "
+            "for the index meta-database"
```

Comment:
```suggestion
            "Absolute path to a JSON file containing the link entries to serve under the /links endpoint"
            "of the index meta-database. These will only be served when using a MongoDB-based backend."
```

Reply:
I'll slightly edit this, as it with your change seems like it's all good until the very end, where I'd prefer to be a bit more upfront 😅

Reply:
Is it a "MongoDB collection" though? I think not... it's just some JSON

Reply:
Hmm... fair enough. It's representing the collection though and can be considered a sort of "db dump", right?